### PR TITLE
Patch 2

### PR DESCRIPTION
**Pull request by syed-perisync**

### CODE REVIEW

File: DCA632FFFE2828A0.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+	"gateway_conf": {
+		"gateway_ID": "DCA632FFFE2828A0",
+		"servers": [ { "server_address": "eu1.cloud.thethings.network", "serv_port_up": 1700, "serv_port_down": 1700, "serv_enabled": true } ],
+		"ref_latitude": 12.972442,
+		"ref_longitude": 77.580643,
+		"ref_altitude": 920,
+		"contact_email": "",
```
Comment (gonzalocasas) [inline DCA632FFFE2828A0.json:8]: please add an email

**Review by gonzalocasas — CHANGES_REQUESTED** [review]

File: DCA632FFFE2828A0.json (hunk 1 above)
Comment (Dndptvvxcn) [reply DCA632FFFE2828A0.json:8]: arroncarson1@gmail.com

Comment (gonzalocasas) [reply DCA632FFFE2828A0.json:8]: Not you, the author of the PR. And not in a comment, as a commit. Stop posting non-sense @Dndptvvxcn

**Review by Dndptvvxcn — CHANGES_REQUESTED** [review]
Run

**Comment by gonzalocasas** [comment]
Close due to inactivity

**Pull request closed without merging**